integration-test-common/src/test/java/org/apache/gravitino/integration/test/MiniGravitino.java

```diff
@@ -78,11 +84,25 @@ public MiniGravitino(MiniGravitinoContext context) throws IOException {
     mockConfDir.mkdirs();
   }
 
-  private void removeAuxRestConfiguration(Properties properties) {
-    // Disable Iceberg REST service
-    properties.remove(
-        AuxiliaryServiceManager.GRAVITINO_AUX_SERVICE_PREFIX
-            + AuxiliaryServiceManager.AUX_SERVICE_NAMES);
+  private void removeAuxRestConfiguration(Properties properties, String serviceToRemove) {
+    String value =
+        properties.getProperty(
+            AuxiliaryServiceManager.GRAVITINO_AUX_SERVICE_PREFIX
+                + AuxiliaryServiceManager.AUX_SERVICE_NAMES);
+    if (value != null && !value.isEmpty()) {
+      List<String> serviceNames = COMMA.splitToList(value);
```

Comment:
Will this `splitToList` also do the `trim` to avoid some invisible characters like white space?

Reply:
yes. Here is the definition of `COMMA`:
```java
private static final Splitter COMMA = Splitter.on(",").omitEmptyStrings().trimResults();
```